### PR TITLE
[Change] Moved some scripts to client/server only

### PR DESCRIPTION
**Pull request by Vam-Jam**
Commits have the list of files changed, more will be added as I find them.

The reason for this is because of an engine PR which will means the server/client will no longer attaches scripts they don't run to blobs/crules, which might result in a small performance improvement in some situations.

### CODE REVIEW

**Comment by Harrison-Miller** [comment]
will need to wait for engine PR to be in a good state. And will need to double check these files as some funky stuff happens in a lot of scripts that might be important for either side even if it doesn't seem it.

**Comment by Vam-Jam** [comment]
Will work on fixing the engine PR today and will double check the script files

**Comment by Harrison-Miller** [comment]
@Vam-Jam can you fix the merge conflict.

**Comment by Vam-Jam** [comment]
Done

File: Entities/Characters/Archer/Archer.cfg [hunk 1]
```diff
@@ -183,8 +183,6 @@ $name                                             = archer
 													ClimbArrows.as;
 													ClimbTree.as;
 													Stomp.as;
-													RunnerHead.as;
```
Comment (Harrison-Miller) [inline Entities/Characters/Archer/Archer.cfg:186]: Why are we removing scripts?

Comment (Harrison-Miller) [reply Entities/Characters/Archer/Archer.cfg:186]: sprite only hooks - got it.

**Comment by Harrison-Miller** [comment]
will need testing before release.

**Pull request merged**